src/utils/CMakeLists.txt

```diff
@@ -33,10 +33,10 @@ exago_add_library(
 # works -Wpedantic for CUDA compile options added a significant amount of new
 # errors, so leaving out for now
 set(CUDA_COMPILE_OPTIONS
-    "$<$<AND:$<COMPILE_LANGUAGE:CUDA>,$<CONFIG:DEBUG>>:SHELL:--compiler-options -Werror,-Wall,-Wextra>"
+    "$<$<AND:$<COMPILE_LANGUAGE:CUDA>,$<CONFIG:DEBUG>>:SHELL:--compiler-options -Wall,-Wextra>"
 )
 set(CXX_COMPILE_OPTIONS
-    "$<$<AND:$<COMPILE_LANGUAGE:CXX>,$<CONFIG:DEBUG>>:-Werror;-Wall;-Wextra;-Wpedantic>"
+    "$<$<AND:$<COMPILE_LANGUAGE:CXX>,$<CONFIG:DEBUG>>:-Wall;-Wextra;-Wpedantic>"
 )
```

Comment:
This was originally added by me to repro xSDK CI failures that we see with other compilers. I would love to keep them in as many builds as possible to ensure that our builds are truly cross-platform.

I can limit this to only a few CI platforms if that helps, but I don't see how this would solve any issues by removing.

Reply:
Can I just PR into this with the fix? It's like 5 lines total

Reply:
Reverting these changes in favor of #68 is my suggestion here

Reply:
On the other hand we can just remove these changes here and merge atomic changes one at a time

Reply:
Yup, please revert this change and merge the branch if the tests pass.